### DISCUSSION TITLE
134

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#66] [General]
# 潮流周刊第134期 - 看人游泳

封面来源于上周在水库看到有人牵一个 🎈 直接一人游着泳，水中一环一环还挺好看，有点儿感觉。

https://weekly.tw93.fun/posts/134-%E7%9C%8B%E4%BA%BA%E6%B8%B8%E6%B3%B3/

**Comment by TheBeacon1** [comment]
推荐一个用了3年的导航页-青柠(https://limestart.cn/),再推荐一个大学同学自己做的导航页,包含非常多的优质网站推荐-花森(http://n.huasenjio.top/#/home)

**Reply by tw93** [reply]
谢谢推荐

**Comment by zimiyu** [comment]
这个 AI 工具集导航 套路有点过头，站内跳转一步就算了，还要去bing搜索加上自己的推广代码转一圈。

**Comment by ugfly1210** [comment]
想问下浏览器主题是啥= =

**Reply by tw93** [reply]
edge 默认的主题，不过是beta版本

**Comment by curly210102** [comment]
Jellylade 误写成 Jellulade 了，话说是不是可以 Pake 化？

**Reply by tw93** [reply]
谢谢提醒，已经更新，万物都可pake化
